doc/metric_spec.md

```diff
@@ -0,0 +1,111 @@
+# Metrics specification:
+
+This is a specification of metrics exposed by the ha_cluster exporter.
+
+All metrics from the exporter start with the prefix `ha_cluster`
+
+Below you have a complete specification, ordered by component.
+
+1. [pacemaker](#pacemaker)
+2. [drbd](#drbd)
+3. [sbd](#sbd)
+4. [corosyncl](#corosync)
+
+# Pacemaker 
+
+The Pacemaker cluster metrics are atomic metrics and represent and updated snapshot of the HA cluster, retrieved fetching the XML CIB of pacemaker.
+
+Some of the pacemaker metrics like `ha_cluster_node_resources` and `ha_cluster_nodes` metrics with labels share a common trait:
+ 
+they can be either set to `1` or they are absent, this is because they track the real state of the cluster resources monitored.
+
+1. [ha_cluster_node_resources](#ha_cluster_node_resources)
```

Comment:
It's occurring to me as I'm reading it in this context, that resources belong to cluster. How about just `ha_cluster_resources`?

Reply:
https://github.com/ClusterLabs/ha_cluster_exporter/issues/61